Tests/PackageLoadingTests/PD_4_2_LoadingTests.swift

```diff
@@ -864,6 +864,7 @@ class PackageDescription4_2LoadingTests: PackageDescriptionLoadingTests {
                     identityResolver: identityResolver,
                     fileSystem: localFileSystem,
                     observabilityScope: observability.topScope,
+                    delegateQueue: .global(),
```

Comment:
Just curious:  what's the reason that `sharedConcurrent` is sometimes used and sometimes `global()`?

Reply:
this was an unrelated change that I removed now and will add in a separate PR